man/fi_domain.3.md

```diff
@@ -375,6 +378,15 @@ are defined.
   manual progress may still need application assistance to process
   received operations.
 
+*FI_PROGRESS_CONTROL_UNIFIED*
+: This progress model indicates that the user will synchronize progressing the
+  data and control operations themselves (i.e.. this makes control interface NOT
+  thread safe). It is only valid for control progress (not data progress). If
+  the control progress is set to FI_PROGRESS_CONTROL_UNIFIED, the data progress
+  must be set to FI_PROGRESS_MANUAL. Setting control=FI_PROGRESS_CONTROL_UNIFIED,
```

Comment:
I don't think this restriction is needed.  Progress can still be auto, and the provider can decide how to deal with that.

Reply:
So you are saying both `data_progress == AUTO && control_progress == UNIFIED` should somehow work?  I think a better way of approaching that situation is `data_progress == auto && control_progress == auto` and letting the provider figure out how to deal with that.

I don't see how it is possible for  `data_progress == AUTO && control_progress == UNIFIED` to be a valid configuration.

Reply:
control = UNIFIED still means that calls to the control and data APIs are serialized by the app.  Setting both to AUTO doesn't convey that.  Do I expect an app to use that combo?  No.  But that's also true for control=manual & data=auto, which is still an allowed API option.

If a provider offloads everything, control=UNIFIED and data=AUTO may be a valid pairing for it to return.  That's probably what the UDP provider can support and still be lockless.

Reply:
Removed the sentence, and the if check in util_domain.c